### PR TITLE
Add 3 new judges to Asylum Support Tribunal

### PR DESCRIPTION
**Pull request by chao-xian**
[Via Zendesk](https://govuk.zendesk.com/agent/tickets/4144714)

## Screenshots

Screenshot at Jul 09 3-26-30 pm
Screenshot at Jul 09 3-27-00 pm
Screenshot at Jul 09 3-27-20 pm

### CODE REVIEW

File: lib/documents/schemas/asylum_support_decisions.json [hunk 1]
```diff
@@ -160,6 +160,10 @@
           "label": "Bayati, C",
           "value": "bayati-c"
         },
+		{
+		  "label": "Beg, S",
+		  "value": "beg-s"
+		},
```
Comment (richardTowers) [inline lib/documents/schemas/asylum_support_decisions.json:166]: Probably doesn't matter, but it looks like you've got a tabs vs. spaces inconsistency here.

Comment (chao-xian) [reply lib/documents/schemas/asylum_support_decisions.json:166]: GAH!

File: lib/documents/schemas/asylum_support_decisions.json [hunk 2]
```diff
@@ -160,6 +160,10 @@
           "label": "Bayati, C",
           "value": "bayati-c"
         },
+    {
+      "label": "Beg, S",
+      "value": "beg-s"
+    },
```
Comment (richardTowers) [inline lib/documents/schemas/asylum_support_decisions.json]: I should probably not be such a :neckbeard: about this, but it looks like you want 8 spaces, not 4...

```suggestion
        {
          "label": "Beg, S",
          "value": "beg-s"
        },
```

Happy to approve it anyway though.

**Review by richardTowers — APPROVED** [review]

File: lib/documents/schemas/asylum_support_decisions.json (hunk 2 above)
Comment (chao-xian) [reply lib/documents/schemas/asylum_support_decisions.json]: Not pedantry - holding me up to the right standards, and thanks!

**Pull request merged**